components/forms/conditionOfJoining/CojView.tsx

```diff
@@ -17,7 +17,7 @@ const CojView: React.FC = () => {
   const signingCoj = store.getState().user.signingCoj;
   const progName = store.getState().user.signingCojProgName;
   const signedDate = store.getState().user.signingCojSignedDate;
-  const canEdit = !!signedDate;
+  const canEdit = signedDate === null ? true : false;
```

Comment:
Since the ternary is just setting true/false we can simply this to just `const canEdit = signedDate === null;`